pkgs/games/ecwolf/default.nix

```diff
@@ -45,6 +61,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
+    broken = stdenv.isDarwin;
```

Comment:
broken usually goes last in meta. That it was first was a mistake from a treewide change. Also a short comment why is usually very helpful for people trying to fix it.

Reply:
I just submitted #212871 which addresses this issue.